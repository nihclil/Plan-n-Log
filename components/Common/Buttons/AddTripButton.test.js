import AddTripButton from "./AddTripButton";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <>
        <AddTripButton />
      </>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
