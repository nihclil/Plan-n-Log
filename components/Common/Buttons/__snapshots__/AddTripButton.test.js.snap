// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdnyFh ktaMfq"
>
  <a
    href="/plan"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="sc-gtsqUy dDJlsa"
    >
      <img
        className="sc-dlniIP"
        src="/iconmonstr-plus-6-24.png"
      />
      <span
        className="sc-hKFymg gtAkQk"
      >
        Add a trip
      </span>
    </div>
  </a>
</div>
`;
